frontend/src/DiscoverMovie/Posters/DiscoverMoviePosters.js

```diff
@@ -132,6 +132,7 @@ class DiscoverMoviePosters extends Component {
         });
       }
     }
+
```

Comment:
Stray new line

Reply:
That was me removing the discover stuff (as it doesn't need the scroll restoration)!